src/uu/realpath/src/realpath.rs

```diff
@@ -58,6 +64,20 @@ pub fn uu_app() -> App<'static, 'static> {
     App::new(executable!())
         .version(crate_version!())
         .about(ABOUT)
+        .arg(
+            Arg::with_name(OPT_LOGICAL)
+                .short("L")
+                .long(OPT_LOGICAL)
+                .help("resolve '..' components before symlinks")
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name(OPT_PHYSICAL)
+                .short("P")
+                .long(OPT_PHYSICAL)
+                .help("resolve symlinks as encountered (default)")
+                .takes_value(false),
+        )
```

Comment:
These options should override each other (i.e. so that only the latest of the two counts). You can use `.overrides_with()` for that.

Reply:
Thanks, I'll add that.